modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -223,22 +244,24 @@ private static int getLogFileSequenceNumber() {
 
         siddhiAppContextName = siddhiAppContext.getName();
 
-        if (attributeExpressionLength == 2) {
+
+        if (attributeExpressionLength == 2 || attributeExpressionLength == 3 ||
+                attributeExpressionLength == 4) {
+
```

Comment:
Use a loop with switch case for each attribute validation, for higher readabilty

Reply:
I think using switch case may incur some redundant code. Is this new commit acceptable?
https://github.com/wso2/product-sp/pull/855/commits/90cdfba07f986f7d84f55a3e4133f0cefe1799cc